### ISSUE TITLE
no-node-access reports issue when RTL's fireEvent object is reexported from another package or file

### ISSUE THREAD

**Issue by WesSouza** [testing-library/eslint-plugin-testing-library#1051]
### Have you read the Troubleshooting section?

Yes

### Plugin version

v7.6.2

### ESLint version

v9.31.0

### Node.js version

v22.11.0

### Bug description

When importing `fireEvent` from a different package than directly from '@testing-library/react', `no-node-access` complains that function calls should not directly access a node:

```js
// @/testing/index.ts
export * from '@testing-library/react';
```

```js
// someTest.test.ts
import { fireEvent, render, screen } from '@/testing';

// ...

render(something);
const button = await screen.findByTestId('some-button');
fireEvent.click(button);
//          ^ - Avoid direct Node access. Prefer using the methods from Testing Library. eslint testing-library/no-node-access 

```

### Steps to reproduce

1. Create a file or package with `export * from '@testing-library/react';`
2. On a test file, replace the import from '@testing-library/react' with that file path or package
3. Have a test calling an event from `fireEvent`, such as `fireEvent.click`
4. `no-node-access` triggers

### Error output/screenshots

Avoid direct Node access. Prefer using the methods from Testing Library. eslint testing-library/no-node-access

### ESLint configuration

```js
import testingLibrary from 'eslint-plugin-testing-library';
// ...

export default [
  // ...
  {
    files: ['**/*.test.*'],
    ...testingLibrary.configs['flat/react'],
  },
];
```

### Rule(s) affected

no-node-access

### Do you want to submit a pull request to fix this bug?

No

**Comment by WesSouza** [comment]
This looks related to #1048, but I'm not sure that would fix this issue.

**Comment by y-hsgw** [comment]
@WesSouza 
I believe enabling [Aggressive Reporting](https://github.com/testing-library/eslint-plugin-testing-library?tab=readme-ov-file#testing-libraryutils-module) might resolve the issue. Could you please take a look?

```ts :diff
import testingLibrary from 'eslint-plugin-testing-library';
// ...

export default [
  // ...
  {
    files: ['**/*.test.*'],
    ...testingLibrary.configs['flat/react'],
    settings: {
+      'testing-library/utils-module': 'testing',
    },
  },
];

**Comment by WesSouza** [comment]
Enabling this seems to cause a side effect on other rules, for example `testing-library/no-render-in-lifecycle` stops working (we have comments disabling it on some places that started to error as below)

```
   95:7   error  Unused eslint-disable directive (no problems were reported from 'testing-library/no-render-in-lifecycle')
```

**Comment by y-hsgw** [comment]
Closed since it has been resolved in https://github.com/testing-library/eslint-plugin-testing-library/pull/1043.
